From bd149a210da9fec4db07f3c54232b0923401dbc2 Mon Sep 17 00:00:00 2001
Message-Id: <bd149a210da9fec4db07f3c54232b0923401dbc2.1415496071.git.chang-joon.lee@intel.com>
In-Reply-To: <aa0c58fe046f9d00f3d381a75f6169b3f6d58c94.1415496071.git.chang-joon.lee@intel.com>
References: <aa0c58fe046f9d00f3d381a75f6169b3f6d58c94.1415496071.git.chang-joon.lee@intel.com>
From: Brad Volkin <bradley.d.volkin@intel.com>
Date: Wed, 4 Jun 2014 10:56:49 -0700
Subject: [PATCH 7/8] Revert "REVERTME [VPG]: drm/i915: Disable command parser
 by default"

This reverts commit c7c209c0bc891ba12251e7981268ff1418bf00b1.

For: APDEV-870
Change-Id: If4d90073b791cf89a8191ddfe75190a0f7df1c1e
Signed-off-by: Brad Volkin <bradley.d.volkin@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 3b9ac90..5d4ec13 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -49,7 +49,7 @@ struct i915_params i915 __read_mostly = {
 	.reset = true,
 	.invert_brightness = 0,
 	.disable_display = 0,
-	.enable_cmd_parser = 0,
+	.enable_cmd_parser = 1,
 	.disable_vtd_wa = 0,
 	.drrs_interval = 2000,
 	.use_mmio_flip = 0,
-- 
1.7.9.5

